src/main/java/xyz/aadev/generitech/common/tileentities/power/TileEntityPower.java

```diff
@@ -81,26 +78,27 @@ public long powerStored() {
     public void update() {
         BlockPos pos = getPos();
         World worldIn = getWorld();
-
+        if (machineTier == null) {
+            machineTier = MachineTier.byMeta(getBlockMetadata());
+        }
         if (fuelRemaining != 0) fuelRemaining--;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00122)